Source/Core/DolphinQt2/Settings/GeneralPane.cpp

```diff
@@ -0,0 +1,116 @@
+#include "DolphinQt2/Settings/GeneralPane.h"
+
+#include <QCheckBox>
+#include <QComboBox>
+#include <QGroupBox>
+#include <QHBoxLayout>
+#include <QLabel>
+#include <QString>
+#include <QVBoxLayout>
+#include <QWidget>
+
+#include "Core/ConfigManager.h"
+
+GeneralPane::GeneralPane(QWidget* parent) : QWidget(parent)
+{
+  CreateLayout();
+  ConnectLayout();
+
+  LoadConfig();
+}
+
+void GeneralPane::CreateLayout()
+{
+  m_main_layout = new QVBoxLayout;
+  // Create layout here
+  CreateBasic();
+  CreateAdvanced();
+
+  m_main_layout->addStretch(1);
+  setLayout(m_main_layout);
+}
+
+void GeneralPane::ConnectLayout()
+{
+  connect(m_combobox_language, static_cast<void (QComboBox::*)(int)>(&QComboBox::activated),
+          [=](int index) { GeneralPane::SaveConfig(); });
+
+  // Advanced
+  connect(m_checkbox_force_ntsc, &QCheckBox::clicked, this, &GeneralPane::SaveConfig);
+  connect(m_checkbox_speed_limit, static_cast<void (QComboBox::*)(int)>(&QComboBox::activated),
+          [=](int index) { GeneralPane::SaveConfig(); });
+}
+
+void GeneralPane::CreateBasic()
+{
+  auto* basic_group = new QGroupBox(tr("Basic Settings"));
+  auto* basic_group_layout = new QVBoxLayout;
+  basic_group->setLayout(basic_group_layout);
+  m_main_layout->addWidget(basic_group);
+
+  auto* language_layout = new QHBoxLayout;
+  basic_group_layout->addLayout(language_layout);
+
+  auto* language_label = new QLabel(tr("Language:"));
+  language_label->setMaximumWidth(150);
+  language_layout->addWidget(language_label);
+
+  m_combobox_language = new QComboBox;
+  // TODO: Support more languages other then English
+  m_combobox_language->addItem(tr("English"));
+  language_layout->addWidget(m_combobox_language);
+}
+
+void GeneralPane::CreateAdvanced()
+{
+  auto* advanced_group = new QGroupBox(tr("Advanced Settings"));
+  auto* advanced_group_layout = new QVBoxLayout;
+  advanced_group->setLayout(advanced_group_layout);
+  m_main_layout->addWidget(advanced_group);
+
+  // Speed Limit
+  auto* speed_limit_layout = new QHBoxLayout;
+  advanced_group_layout->addLayout(speed_limit_layout);
+
+  auto* speed_limit_label = new QLabel(tr("Speed Limit:"));
+  speed_limit_label->setMaximumWidth(150);
+  speed_limit_layout->addWidget(speed_limit_label);
+
+  m_checkbox_speed_limit = new QComboBox;
```

Comment:
`QSlider` works, but it's difficult to get labels. Switching to this implementation for now though.